src/v2/options.ts

```diff
@@ -148,6 +105,11 @@ export interface GlobalOptions {
    * Timeout for the function in sections, possible values are 0 to 540.
    * HTTPS functions can specify a higher timeout.
    * A value of null restores the default of 60s
+   * The minimum timeout for a gen 2 function is 1s. The maximum timeout for a
+   * function depends on the type of function: Event handling functions have a
+   * maximum timeout of 540s (9 minutes). HTTPS and Callable functions have a
+   * maximum timeout of 36,00s (1 hour). TaskQueue functions have a maximum
```

Comment:
Suggest "task queue functions" and "callable functions" unless these are literals.